packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -741,27 +727,25 @@ curl -v -X GET \
 }
 ```
 
-### Add Property to App User Profile Schema
-
+### Add property to App User Profile Schema
 
 <ApiOperation method="post" url="/api/v1/meta/schemas/apps/${instanceId}/default" />
 
-Adds one or more [custom app user profile properties](#app-user-profile-schema-property-object) to the app user schema
+Adds one or more [custom App User Profile properties](#app-user-profile-schema-property-object) to the App User schema
 
-##### Request Parameters
+##### Request parameters
 
 
 | Parameter   | Description                                          | Param Type | DataType                                                       | Required | Default |
 | :----------- | :---------------------------------------------------- | :---------- | :-------------------------------------------------------------- | :-------- | :------- |
 | instanceId  | The id of the target App Instance                    | Path       | String                                                         | TRUE     |        |
```

Comment:
"ID" ... maybe a quick search

Reply:
So, this actually needed to be `id` - because of the way it is represented in the code.